deploy/sophosxg.sh

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
+#
+# This deploy script deploys to a Sophos XG appliance
+# DEPLOY_SOPHOSXG_HOST="<NO DEFAULT - REQUIRED - host:port>"
+# DEPLOY_SOPHOSXG_USER="<NO DEFAULT - REQUIRED - string>"
+# DEPLOY_SOPHOSXG_PASSWORD="<NO DEFAULT - REQUIRED - string>"
+# DEPLOY_SOPHOSXG_NAME="domain"
+# DEPLOY_SOPHOSXG_PFX_PASSWORD="s0ph0sXG"
+# DEPLOY_SOPHOSXG_HTTPS_INSECURE="1"
+
+########  Public functions #####################
+
+#action pfx user password name pfxpass host
+sophosxg_do_req() {
+  # check number of args
+  [ $# -eq 7 ] || return 1
+
+  # set vars
+  _do_req_action="$1"
+  _do_req_pfx="$2"
+  _do_req_user="$3"
+  _do_req_password="$4"
+  _do_req_name="$5"
+  _do_req_pfxpass="$6"
+  _do_req_host="$7"
+
+  # static values - as variables in case these need to change
+  _do_req_boundary="SOPHOSXGPOST"
+  _do_req_certfile="certificate.p12"
+
+  # dont verify certs if config set
+  _do_req_old_HTTPS_INSECURE="${HTTPS_INSECURE}"
```

Comment:
don't need to save _do_req_old_HTTPS_INSECURE

the shell is isolated,  you won't affect others.

Reply:
ack